content/en/docs/ops/diagnostic-tools/multicluster/index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Debugging Virtual Machines
+description: Describes tools and techniques to diagnose issues with Virtual Machines.
```

Comment:
Should this be multicluster instead of VM?

Reply:
right, thanks!